src/main/java/com/conveyal/r5/analyst/Grid.java

```diff
@@ -310,11 +312,22 @@ public void write (OutputStream outputStream) throws IOException {
         out.writeInt(width);
         out.writeInt(height);
         // The rest of the file is 32-bit integers in row-major order (x changes faster than y), delta-coded.
-        for (int y = 0, prev = 0; y < height; y++) {
+        // Delta coding and error diffusion are reset on each row to avoid wrapping.
+        int prev = 0;
+        for (int y = 0; y < height; y++) {
+            // Reset error on each row to avoid diffusing to distant locations.
+            // An alternative is to use serpentine iteration or iterative diffusion.
```

Comment:
Serpentine iteration seemed appealing to me initially. But it's probably not worth the added complexity, and it's not clear that southward bias is preferable to eastward bias.